packages/block-library/src/post-content/edit.js

```diff
@@ -104,25 +105,18 @@ function Content( props ) {
 	);
 }
 
-function Placeholder( { layoutClassNames } ) {
+function ContentPlaceholder( { layoutClassNames } ) {
 	const blockProps = useBlockProps( { className: layoutClassNames } );
 	return (
 		<div { ...blockProps }>
-			<p>
-				{ __(
-					'This is the Content block, it will display all the blocks in any single post or page.'
-				) }
-			</p>
-			<p>
-				{ __(
-					'That might be a simple arrangement like consecutive paragraphs in a blog post, or a more elaborate composition that includes image galleries, videos, tables, columns, and any other block types.'
-				) }
-			</p>
-			<p>
-				{ __(
-					'If there are any Custom Post Types registered at your site, the Content block can display the contents of those entries as well.'
-				) }
-			</p>
+			<Placeholder
+				className="block-editor-media-placeholder"
+				withIllustration={ true }
+			>
+				<p>
+					{ __( 'This is where your post or page content will go.' ) }
```

Comment:
Given the block is also used for custom post types. Restricting this to "post" or "page" could be confusing for those templates. Also, from a user perspective, the template already appears to have content so the current wording could seem a bit odd.

What about something like:

> This is where the dynamic content (for example the blocks from a specific "post" or "page") will go.

Reply:
If this placeholder is in a post-type-specific template, it _could_ be more context-aware (and ideally would be text provided when registering the post type).